### ISSUE TITLE
Set up a license

### ISSUE THREAD

**Issue by CryZe** [CryZe/livesplit-one-desktop#10]
We may potentially want to use GPL here instead of MIT. I'm not sure yet, but we definitely need a license.

**Comment by wooferzfg** [comment]
I think it would make sense to use MIT for any repos that might be used by other projects, such as `livesplit-core` or `splits-io-api`, and GPL for any repos that are standalone apps, such as `livesplit-one` or `livesplit-one-web` (idk if we can change the license of that repo).

**Comment by CryZe** [comment]
Yeah that's mostly what I've been thinking too. We should likely separate the renderer out and license that as MIT then.

**Comment by CartoonFan** [comment]
Hi there! Just curious, but has there been any progress on this? It seems to be the last blocker for #9...

**Comment by CryZe** [comment]
Well the actual blocker is that there's no splits editor, layout editor or any other sort of real GUI yet other than showing the timer itself. So I'd like to get that sorted out first and then we can add a license and move it over. Though I guess we can add a license any time, it's just not our actual main priority.

**Comment by CartoonFan** [comment]
Ah, I see. Thanks for replying!
